.env.example

```diff
@@ -116,15 +116,31 @@ NEXT_PUBLIC_FRESHCHAT_HOST=
 # @see https://support.google.com/cloud/answer/6158849#public-and-internal&zippy=%2Cpublic-and-internal-applications
 GOOGLE_LOGIN_ENABLED=false
 
-#   - GOOGLE CALENDAR/MEET/LOGIN
+# Hanko Passkeys Config
```

Comment:
lets mention this lower in the file

Reply:
yeah was unsure about that. wanted to place it below google auth stuff since i did that everywhere else, but agreed, for the env file its a little too high up